src/lib/template/dom-repeat.html

```diff
@@ -253,6 +316,50 @@
       }
     },
 
+    _limitChanged: function(limit) {
+      if (this.items) {
+        this._debounceTemplate(this._render);
+      }
+    },
+
+    _computeFrameTime: function(rate) {
+      return Math.ceil(1000/rate);
+    },
+
+    _initializeChunkCount: function() {
+      if (this.initialCount) {
+        this.limit = this.initialCount;
+        this._chunkCount = parseInt(this.chunkCount, 10) || this.initialCount;
+      }
+    },
+
+    _tryRenderChunk: function() {
+      if (this._chunkCount &&
+          Math.min(this.limit, this._instances.length) < this.items.length) {
+        this.debounce('renderChunk', this._requestRenderChunk);
```

Comment:
comment: debounce required to ensure chunked rendering & data mutation rendering don't cause duplicate rAF's